core/src/main/scala/org/apache/spark/storage/memory/MemoryStore.scala

```diff
@@ -318,19 +291,26 @@ private[spark] class MemoryStore(blockManager: BlockManager, memoryManager: Memo
     blockId.asRDDId.map(_.rddId)
   }
 
+  private def acquireStorageMemory(blockId: BlockId, size: Long): Boolean = {
```

Comment:
Sure, I can move it back. This method is going away anyways in my next patch, though.